cpp/include/experimental/graph_generator.hpp

```diff
@@ -72,13 +72,70 @@ template <typename vertex_t>
 std::tuple<rmm::device_uvector<vertex_t>, rmm::device_uvector<vertex_t>> generate_rmat_edgelist(
   raft::handle_t const& handle,
   size_t scale,
-  size_t edge_factor       = 16,
+  size_t num_edges         = 16,
```

Comment:
Sorry for bugging again, but please delete the default value as well please :-)

Reply:
yup